index.js

```diff
@@ -0,0 +1,28 @@
+require("dotenv").config();
+const aws = require("aws-sdk");
+
+(async function() {
+    try {
+        
+        aws.config.setPromisesDependency();
+
+        aws.config.update({
+            accessKeyId: process.env.ACCESS,
+            secretAccessKey: process.env.SECRET,
+            region: "us-east-1"
+        });
+
+        const s3 = new aws.S3();
+        const response = await s3.listObjectsV2({
+            Bucket: "cru.org-storylines"
```

Comment:
Should probably make this an environment variable as well.

Reply:
Done.